From 4b96bdccece38c3695c45b5109da4859d48a6971 Mon Sep 17 00:00:00 2001
From: Hardik Kantilal Patel <hkpatel@codeaurora.org>
Date: Fri, 13 Feb 2015 17:59:57 +0530
Subject: [PATCH 35/39] wcnss: Avoid writing zero to PMU config register

Avoid writing zero to PMU config register while
taking MUX control.

Change-Id: Ib05cd43a8d3bc4df27e27fef9e1539834d72e77a
CRs-Fixed: 794268
Signed-off-by: Hardik Kantilal Patel <hkpatel@codeaurora.org>
---
 drivers/net/wireless/wcnss/wcnss_wlan.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/wcnss/wcnss_wlan.c b/drivers/net/wireless/wcnss/wcnss_wlan.c
index 4a0382a..43547dc 100644
--- a/drivers/net/wireless/wcnss/wcnss_wlan.c
+++ b/drivers/net/wireless/wcnss/wcnss_wlan.c
@@ -899,7 +899,6 @@ int wcnss_get_mux_control(void)
 		return 0;
 
 	pmu_conf_reg = penv->msm_wcnss_base + PRONTO_PMU_OFFSET;
-	writel_relaxed(0, pmu_conf_reg);
 	reg = readl_relaxed(pmu_conf_reg);
 	reg |= WCNSS_PMU_CFG_GC_BUS_MUX_SEL_TOP;
 	writel_relaxed(reg, pmu_conf_reg);
-- 
2.5.0

